website/static/js/index.js

```diff
@@ -204,9 +205,11 @@ elements.overlay.addEventListener("touchstart", mobileToggleEvent, false);
 window.addEventListener("scroll", handleScroll, false);
 
 elements.colorSchemeSwitch.addEventListener("click", modeSwitch, false);
-const teamArr = Array.from(elements.teamList.getElementsByTagName("li"));
-for (const li of randomShuffle(teamArr)) {
-	elements.teamList.appendChild(li);
+const teamArr = document.querySelectorAll(".team-member-item");
```

Comment:
You can avoid having the `team-member-item` class by using a descendant selector.

```suggestion
const teamArr = document.querySelectorAll(".team-list li");
```